### PR TITLE
fix monitoring certificate

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

File: installer/openshift/roles/getup-certificate-checker/templates/external-certificate-cron.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    spec:
+      template:
+        spec:
+          containers:
+          - name: certificate-external-checker
+            image: getupcloud/check-external-certificate:latest
+            imagePullPolicy: Always
+            env:
+            - name: ENDPOINT
+              value: api.{{ openshift_master_default_subdomain }}
```
Comment (caruccio) [inline installer/openshift/roles/getup-certificate-checker/templates/external-certificate-cron.yaml.j2]: change `api.{{ openshift_master_default_subdomain }} ` to `{{ kubernetes_api_endpoint }}`

Comment (Talits) [reply installer/openshift/roles/getup-certificate-checker/templates/external-certificate-cron.yaml.j2]: ok

**Comment by caruccio** [comment]
@Talits  Looks like `templates/external-certificate-cron.yaml.j2` is a dup fo `files/cronjob-certificate-checker.yaml`. Is it correct?

**Comment by Talits** [comment]
yes is correct, one is a certificate check to internal components

**Comment by Talits** [comment]
[[Monitoramento]Verificação certificados](https://trello.com/c/mQPrXK4F/174-monitoramentoverifica%C3%A7%C3%A3o-certificados)

**Pull request merged**